integration/command.go

```diff
@@ -517,6 +564,56 @@ EOF
 	}
 }
 
+// PodCommandThroughAPI returns a Command that starts a pid with a specified image, command and args
+func PodCommandThroughAPI(podname, image, namespace string, command, commandArgs []string) *K8sCommand {
+	return &K8sCommand{
+		name: fmt.Sprintf("Creating pod %s", podname),
+		runFunc: func(t *testing.T) {
+			k8sClient, err := k8sutil.NewClientsetFromConfigFlags(utils.KubernetesConfigFlags)
+			if err != nil {
+				t.Fatalf("Creating pod (%s): %s", podname, commonutils.WrapInErrSetupK8sClient(err).Error())
+				return
+			}
```

Comment:
It would be nice to create this once and share in all invocations to these functions.